source/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Text.RegularExpressions;
+
+namespace TownOfUs.Extensions
+{
+    public static class StringExtensions
+    {
+        public static string StripHtmlTag(this string input)
+        {
+            return Regex.Replace(input, "<.*?>", string.Empty);
```

Comment:
I’m skeptical of this. It can overzealously remove stuff we don’t want. It makes me wonder if the answer is that we shouldn’t be wrapping it in these tags in the first place.
You might consider modifying it to actually look for matched tags. But HTML isn’t modeled by a FSM; you need a CFG.

Reply:
> we shouldn’t be wrapping it in these tags in the first place

We can use the same method only when importing perhaps?

I don't see how this can affect stuff we want. This tags are mostly used for TextMeshPro's rich text tag and we don't want to use it anywhere outside it. I might look for matched tags (likely from [here](http://digitalnativestudios.com/textmeshpro/docs/rich-text/), but I'll look for better alternative).

Reply:
If any of our text ever contains angle brackets, it would strip those out. I don't think we do that now, but it's not inconceivable.